components/Current-Weather/current-weather.js

```diff
@@ -0,0 +1,26 @@
+import { useEffect, useState } from "react";
+import style from "../../styles/Home.module.css";
+
+const CurrentWeather = ({data}) => {
+const [currentWeather, setCurrentWeather] = useState("");
+
+const properties = data.properties;
+const periods = properties.periods;
+const firstPeriod = periods[0]
+// i want it to show the current weather for my location, the temp, if its sunny or partly cloudy, and maybe the high and low for the day
```

Comment:
I like this idea! Is the data in the `data` object to support this? If so is there anything you're uncertain about implementing these ideas?

Bonus feature we could show different icons for sunny or cloudy or rainy.

Reply:
yes